### PR TITLE
feature (firebolt-driver) engine autostart

### PR DESCRIPTION
**Pull request by Jeseeq**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Issue Reference this PR resolves**
Resolves issue [#4900]

### CODE REVIEW

File: packages/cubejs-testing/src/REQUIRED_ENV_VARS.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const REQUIRED_ENV_VARS: {[key: string]: string[]} = {
     'CUBEJS_DB_USER',
     'CUBEJS_DB_PASS',
     'CUBEJS_DB_NAME',
-    'CUBEJS_FIREBOLT_ENGINE_ENDPOINT'
+    'CUBEJS_FIREBOLT_ENGINE_NAME'
```
Comment (cristipp) [inline packages/cubejs-testing/src/REQUIRED_ENV_VARS.ts:22]: Add `CUBEJS_FIREBOLT_ACCOUNT` here too?

Comment (Jeseeq) [reply packages/cubejs-testing/src/REQUIRED_ENV_VARS.ts:22]: `CUBEJS_FIREBOLT_ACCOUNT` is not really required. Firebolt sdk resolves yours default account if account is not specified. In Firebolt your user may be a member of > 1 account, only then you need to specify account name.

Comment (cristipp) [reply packages/cubejs-testing/src/REQUIRED_ENV_VARS.ts:22]: Fair enough. That is a good point!

For background, the naming here is not that fortunate, the goal of these variables is two fold:
1. Ensure that the auth is correctly set.
2. Copy auth vars into Docker environment, if using Docker-ized cubejs.
I will make a separate PR to automatize 2. based on env var prefixes, so it gets less confusing.

**Comment by cristipp** [comment]
LGTM. Thanks for the contribution!

**Pull request merged**